### ISSUE TITLE
Including a License

### ISSUE THREAD

**Issue by joelmbell** [Vegas-iOS-Developers/LasVegasArtWorkTour#4]
Is it important that include a license? I figured we should consider it because this is an open-source project that is available to the public. 

http://choosealicense.com is a great site that explains the differences.

**Comment by WilliamJones** [comment]
I agree also. Good site.  Which license do you think would be the best option?

**Comment by joelmbell** [comment]
I'm not sure, I need to do a bit of research. I'm thinking since this will be a completed app I'd like to have a license that prohibits someone from just taking this source code and putting the app on the app store to make money for themselves without even modifying it at all. I'm not sure if this is realistic though according to this Quora answer

https://www.quora.com/What-is-the-best-free-for-non-commercial-use-license-to-use-for-an-open-source-project
